modules/nw-routeadvertisements-example.adoc

```diff
@@ -0,0 +1,245 @@
+// Module included in the following assemblies:
+//
+// * networking/route_advertisements/about-bgp-routing.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-examples_{context}"]
+= Examples advertising pod IP addresses with BGP
+
+The following examples describe several different configurations for advertising pod IP addresses with BGP. The external network border router has the `172.18.0.5` IP address.
+
+Each example relies upon the following `FRRConfiguration` object:
+
+.`FRRConfiguration` CR
+[source,yaml]
+----
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: receive-all
+  namespace: openshift-frr-k8s
```

Comment:
You need to add the label you are using in the RouteAdvertisements examples below:
```
...
metadata:
  name: receive-all
  namespace: openshift-frr-k8s
  labels:
    routeAdvertisements: receive-all
...
```